RTQA/iPython/extensions/tagset_gen.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/python3
```

Comment:
This file was in Brian's fork. I have no clue what this file does. I did not made any changes to it. I know what it is used by the magic, since it's one of the arguements to one of the subprocesses that praxi.py starts.